Gifski/EditVideoViewController.swift

```diff
@@ -282,6 +299,20 @@ final class EditVideoViewController: NSViewController {
 		}
 	}
 
+	private func showConversionCompletedAnimationWarningIfNeeded() {
+		// TODO: This function can be removed once NSImageView respects the looping counter in the GIF header.
+		// https://github.com/feedback-assistant/reports/issues/187
+		SSApp.runOnce(identifier: "gifLoopCountWarning") {
+			DispatchQueue.main.async { [self] in
+				NSAlert.showModal(
+					for: view.window,
+					message: "Animated GIF Preview Limitation",
+					informativeText: "The after conversion preview and Quick Look will may not loop as expect, but the exorted file is valid."
```

Comment:
There are 3 typos in this text. I think it also could be more explicit about in what case it may not loop as expected.